worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -154,6 +161,23 @@ namespace RTC
 		this->startSeq = 0;
 	}
 
+	void RtpStreamSend::StorageItemBuffer::Dump()
+	{
+		for (size_t i{ 0 }; i < this->buffer.size(); i++)
```

Comment:
Why not using iterator API instead here?

Reply:
Just to be able to print the index if the `StorageItem` is `nullptr.